src/lib/components/ChatInputField.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import { InputField } from "virtualized-chat";
+import "./quill.snow.css";
```

Comment:
ok, I guess this is because the VM and global css's. isn't it?

Reply:
yes, solution for now